src/api/setup/mnemonic.js

```diff
@@ -0,0 +1,3 @@
+import { generateMnemonic } from "bip39";
+
+export const generateMnemonic = () => generateMnemonic();
```

Comment:
We should import this function on the core file so when we do new HD... we pass this value as mnemonic

Reply:
Do you mean import the function from bip39 on the core file or import this function on the core file? I prefer the latter because it leaves room to cleanly add more mnemonic functions into mnemonic.js.